tests/base/src/test/java/org/keycloak/tests/admin/user/UserUpdateTest.java

```diff
@@ -0,0 +1,340 @@
+package org.keycloak.tests.admin.user;
+
+import jakarta.ws.rs.BadRequestException;
+import jakarta.ws.rs.ClientErrorException;
+import jakarta.ws.rs.NotFoundException;
+import org.junit.Assert;
```

Comment:
There is also a forgotten JUnit4 Assert import

Reply:
updated, thanks